Automattic/jetpack
# Contact Form: Show child blocks in the Block Library

Fixes #26884

#### Changes proposed in this Pull Request:

Uses the Forms child blocks to create simple variations to allow form fields to be searchable in the Block Library.
Introduces the flag `hiddenFromPicker` to allow some variations to be hidden from the Form block placeholder.

#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Open the Page/Post editor and click the "+" icon at the top-left corner to open the Block Library
* You should be able to search for any Form child block. E.g. `Text`, `Email`, `URL`, `Name`, etc.
* By selecting any of the blocks, a Form block should be added to the Post/Page containing the selected child block and a submit button
